hypervisor/common/hypercall.c

```diff
@@ -301,6 +303,10 @@ int32_t hcall_create_vm(struct acrn_vcpu *vcpu, struct acrn_vm *target_vm, uint6
 
 	}
 
+	if (((ret != 0) || (cv.vmid == ACRN_INVALID_VMID)) && (!is_static_vm_configured(target_vm))) {
+		memset(vm_config->name, 0U, MAX_VM_NAME_LEN);
+	}
```

Comment:
This operation is down in VM shutdown time. Why we need to do again here?

Reply:
before calling hcal_create_vm,   it has copied vm name and cpu_affinity to vm_config inside parse_target_vm,
if there is wrong configuration maybe create vm will failed, and it will not call shutdown_vm in this case, then clear the name.
it will use vm_config inside hcall_create_vm, then copy vm name and cpu_affinity to vm_config in parse_target_vm